pages/accounts/profile.py

```diff
@@ -222,7 +222,104 @@ class InactiveOption(Region):
     class PopupConsole(Region):
         """Popup console interaction."""
 
-        _root_locator = (By.CSS_SELECTOR, '.modal-content')
+        # _root_locator = (By.CLASS_NAME, 'modal-content')
+        _users_locator = (By.LINK_TEXT, 'Users')
+        _misc_locator = (By.LINK_TEXT, 'Misc')
+        _links_locator = (By.LINK_TEXT, 'Links')
+        _full_console_locator = (By.PARTIAL_LINK_TEXT, 'Console')
+
+        @property
+        def misc(self):
+            self.find_element(*self._misc_locator).click()
+            return self.Misc(self)
+
+        @property
+        def users(self):
+            self.find_element(*self._users_locator).click()
+            return self.Users(self)
+
+        @property
+        def links(self):
+            self.find_element(*self._links_locator).click()
+            return self.Links(self)
+
+        @property
+        def full_console(self):
+            self.find_element(*self._full_console_locator).click()
+
+        class Misc(Region):
+            _task_locator = (By.PARTIAL_LINK_TEXT, 'Run')
+            _security_locator = (By.PARTIAL_LINK_TEXT, 'SecurityTransgression')
+            _routing_locator = (By.PARTIAL_LINK_TEXT, 'RoutingError')
+            _controller_locator = (By.PARTIAL_LINK_TEXT, 'UnknownController')
+            _action_locator = (By.PARTIAL_LINK_TEXT, 'UnknownAction')
+            _template_locator = (By.PARTIAL_LINK_TEXT, 'Template')
+            _not_yet_implemented_locator = (By.PARTIAL_LINK_TEXT, 'NotYetImplemented')
+    
+
+            def task_locate(self):
```

Comment:
Stylistically, these need to be formatted as:
`
try:
    return self.find_element(*<locator>)
except NoSuchElementException:
    return None
`
It will either return the element or it won't. (Boolean None equates to False)